xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -648,16 +648,43 @@ AVDictionary *CDVDDemuxFFmpeg::GetFFMpegOptionsFromInput()
       std::string name = it->first; StringUtils::ToLower(name);
       const std::string &value = it->second;
 
-      if (name == "seekable")
-        av_dict_set(&options, "seekable", value.c_str(), 0);
-      else if (name == "user-agent")
+      if (name == "user-agent")
       {
         av_dict_set(&options, "user-agent", value.c_str(), 0);
         hasUserAgent = true;
       }
+      
+      // these are the officially supported headers from ffmpeg
+      else if (name == "seekable" || name == "chunked_post" || name == "content_type" || name == "metadata" || name == "offset" ||
```

Comment:
this is not what I suggested. addons are not supposed to set ffmpeg options directly because they don't influence how Kodi would open the stream. go through the list of http headers. then check it you can set the option directly by av_dict_set or you need to set it in headers.